gapis/server/server.go

```diff
@@ -914,6 +914,13 @@ func (s *server) GetTimestamps(ctx context.Context, req *service.GetTimestampsRe
 	return replay.GetTimestamps(ctx, req.Capture, req.Device, req.LoopCount, h)
 }
 
+func (s *server) GpuProfile(ctx context.Context, req *service.GpuProfileRequest) (*service.GpuProfileResponse, error) {
```

Comment:
This should return `(service.ProfilingData, error)` instead.

Reply:
See https://github.com/pmuetschard/gapid/commit/e8909ff5093f12ff5d25912c55aa6883d0bad6e2